packages/components/src/presentation-components/tree/controlled/PresentationTreeNodeRenderer.tsx

```diff
@@ -35,7 +35,35 @@ export function PresentationTreeNodeRenderer(props: PresentationTreeNodeRenderer
   const nodeItem = props.node.item;
 
   if (isPresentationInfoTreeNodeItem(nodeItem)) {
-    return <TreeNode isLeaf={true} label={<Text isMuted>{nodeItem.message}</Text>} level={props.node.depth} isHoverDisabled={true} />;
+    return (
+      <TreeNode
+        isLeaf={true}
+        label={
+          <span>
+            <Text isMuted className="info-tree-node-item">
+              {nodeItem.type === InfoTreeNodeItemType.ResultSetTooLarge && (
+                <span>
+                  <span>Provide </span>
+                  <UnderlinedButton
+                    onClick={() => {
+                      if (nodeItem.parentId !== undefined) {
```

Comment:
Probably shouldn't show button if there is no `parentId`

Reply:
Resolved in private